cypress/integration/page_header_spec.js

```diff
@@ -10,7 +10,10 @@ describe('Page Headers', () => {
   it('navigates to the development page', () => {
     cy.contains('a', 'Development').click();
     cy.url().should('match', /development/);
-    cy.get('.docSearch-content h1').invoke('text').invoke('trim').should('equal', 'Development');
+    cy.get('.docSearch-content h1').should(($h1) => {
```

Comment:
Without this change, Cypress would look at the selector before the page had a chance to re-render with the updated body content after the navigation. Cypress would see the old content and fail. Though not obvious, this syntax basically makes Cypress await here.